package.json

```diff
@@ -49,7 +51,7 @@
     "ember-cli-babel": "^6.0.0",
     "ember-cli-htmlbars": "^1.3.0",
     "ember-cli-version-checker": "^1.1.6",
-    "fastboot-filter-initializers": "0.0.1"
+    "fastboot-filter-initializers": "^0.0.2"
```

Comment:
Caret here does nothing, we should bump to 0.1.0 (upstream) so that we can use caret deps...